test/unit/content-src/components/DiscoveryStreamComponents/DSLinkMenu.test.jsx

```diff
@@ -13,6 +13,28 @@ describe("<DSLinkMenu>", () => {
     wrapper = shallowWithIntl(<DSLinkMenu {...ValidDSLinkMenuProps} />);
   });
 
+  it("on Menu Update", () => {
+    const dsLinkMenu = mountWithIntl(<DSLinkMenu />);
+    dsLinkMenu.setState({showContextMenu: true});
+
+    dsLinkMenu.instance().onMenuUpdate(false);
+    dsLinkMenu.update();
+
+    // Should check grand parent element is missing active, how?
```

Comment:
@Mardak is there any way to get div (e.g.  .ds-card) containing DSLinkMenu here. Thought should ask you before moving test to check for "active" and "last-item" class in DSCard component

Reply:
The behavior of DSLinkMenu modifying ancestors does look strange, but ignoring that, you can use `dsLinkMenu.getDOMNode().parentNode.classList` to check if "active" is in there (for the "onMenuShow" test). For here "onMenuUpdate", you can do the opposite by adding the class name after mounting then checking it's removed.